src/components/GroupList/GroupItem.js

```diff
@@ -0,0 +1,61 @@
+/* @flow */
+
+import React from 'react'
+import Divider from 'material-ui/Divider'
+import Avatar from 'material-ui/Avatar'
+import FileFolder from 'material-ui/svg-icons/file/folder'
+import { ListItem } from 'material-ui/List'
+import type { GroupType } from 'ducks/projects'
+
+export type Props = {
+  group: GroupType,
+  inset: number,
+  clickHandler: Function,
+}
+
+
+const GroupItem = (props : Props) => {
+  const {
+    group,
+    clickHandler,
+    inset,
+  } = props
+
+  const {
+    name,
+    description,
+    id,
+  } = group
+
+  function touchTapEventHandler() {
+    if (clickHandler) {
```

Comment:
> stateless functional components wrapped in a class internally

That will probably change in the future. ; ) But yeah, I mostly like the pure functions for the clarity it gives - there can be no life cycle or class methods.

 > I would recommend here rather class.

Yes, let us do that when we have functions that needs to be bound.